mock-server/datastore-server.ts

```diff
@@ -37,25 +40,38 @@ const packageDefinition = protoLoader.loadSync(PROTO_PATH, {
 });
 const descriptor = grpc.loadPackageDefinition(packageDefinition);
 
-/**
- * Implements the runQuery RPC method.
- */
-function grpcEndpoint(
-  call: {},
-  callback: (arg1: string | null, arg2: {}) => {},
-) {
-  // SET A BREAKPOINT HERE AND EXPLORE `call` TO SEE THE REQUEST.
-  callback(null, {message: 'Hello'});
```

Comment:
This grpcEndpoint function has just been moved inside the only place it is used.